source/gui.cpp

```diff
@@ -571,6 +571,13 @@ void reshade::runtime::draw_overlay_menu_home()
 	if (!_effects_enabled)
 		ImGui::Text("Effects are disabled. Press '%s' to enable them again.", input::key_name(_effects_key_data).c_str());
 
+	if (!std::filesystem::exists(_screenshot_path))
+	{
+		ImGui::PushStyleColor(ImGuiCol_Text, COLOR_RED);
+		ImGui::Text("Unable to save screenshots: directory '%s' doesn't exist!", _screenshot_path.string().c_str());
```

Comment:
Good idea =) Some improvements:
- This can be simplified with `ImGui::TextColored(...);`.
- Also, it'd change the text to `Unable to save screenshots because path doesn't exist: %s` so that it is readable even when the UI is very narrow (on small resolutions).
- If you use `u8string()` instead of `string()` ImGui will be able to render text for paths with unicode characters.